components/scream/CMakeLists.txt

```diff
@@ -110,19 +112,30 @@ option (SCREAM_RRTMGP_DEBUG "Turn on extra debug checks in RRTMGP" ${SCREAM_DEBU
 enable_testing()
 include(CTest)
 
-set (CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
+if (CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
```

Comment:
Is `-cpp` not needed with Intel's ifort?

Reply:
This fix was proposed for standalone scream build for scorpio by Jayesh. I do not know what the right fix is here.

Reply:
> This fix was proposed for standalone scream build for scorpio by Jayesh. I do not know what the right fix is here.

Was this fix for Crusher?

Reply:
Only for standalone scream build on crusher.

Reply:
Ok. I'm a little worried about the specificity of this, so for safety, would you wrap this in HIP_BUILD or whatever HIP-specific variable makes sense in this context?

Reply:
i will do
```
if (NOT HIP_BUILD)
  set (CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
endif()
```
as that would revert it to default for all other cases.

Reply:
Your version is fine.